### ISSUE TITLE
Definition of 'GO:0097242 amyloid-beta clearance' update

### ISSUE THREAD

**Issue by BarbaraCzub** [geneontology/go-ontology#13944]
Current definition of 'GO:0097242 amyloid-beta clearance':
"The process in which amyloid-beta is removed from the brain via receptors." 

Amyloid-beta clearance by transcytosis across the blood-brain barrier has probably been studied the most, however, not all 'clearance' of amyloid-beta involves transcytosis (i.e. removal from _the brain_). Sometimes amyloid-beta gets 'cleared' locally by intracellular degradation, e.g. the autophagy-lysosomal pathway. 
Reference: LRP promotes endocytosis and degradation, but not transcytosis, of the amyloid-beta peptide in a blood-brain barrier in vitro model.
(PMID:18289866 PMCID:PMC2376120)
http://europepmc.org/abstract/MED/18289866 

Additionally, PMID:19406747 shows that some cell surface receptors are involved in reducing amyloid-beta levels extracellularly: "Unlike insulin, IGF-1 did not increase the ADDL immunoreactivity in the lysate fraction of cortical neurons (Fig. 3A-4). It caused, however, a profound reduction in ADDL reactivity in the medium (Fig. 3, A and D-4), which was also blocked by AG1024 (Fig. 3A-6). (...) Unlike IR [Insulin receptor], activation of which mediates ADDL attachment to cells, _IGF-1R [Insulin-like growth factor 1 receptor] activation appears to clear ADDLs using an extracellular mechanism_."

Consequently, could we please update this definition as follows: 
"The process in which amyloid-beta is removed from extracellular brain regions by mechanisms involving cell surface receptors." 

Perhaps in addition more specific child terms could be created to distinguish between the different mechanisms.

> NTR1: amyloid-beta clearance by transcytosis (parents: 'GO:0097242 amyloid-beta clearance' and 'GO:0045056 transcytosis').
Suggested Def.: "The process in which amyloid-beta is removed from extracellular brain regions by cell surface receptor-mediated endocytosis, followed by transcytosis across the blood-brain barrier." 

> NTR2: amyloid-beta clearance by cellular catabolic process (parents: 'GO:0097242 amyloid-beta clearance' and 'GO:0044248 cellular catabolic process')
Suggested Def.: "The process in which amyloid-beta is removed from extracellular brain regions by cell surface receptor-mediated endocytosis, followed by intracellular degradation." 

> NTR3 amyloid-beta clearance by extracellular catabolic process (parents: 'GO:0097242 amyloid-beta clearance' and 'GO:0009056 catabolic process')
Suggested Def.: "The process in which amyloid-beta is cleared by extracellular degradation mechanisms mediated by cell-surface receptors."
(The mechanism revealed in Figure 8 in PMID-19406747 implicates the secreted neprilysin and insulin-degrading enzyme (IDE) into extracellular amyloid-beta clearance).
 
Thanks, 
Barbara

**Comment by vanaukenk** [comment]
Hi @BarbaraCzub 

I will go ahead and add these new terms, and update the existing parent term definition.  Thanks for the definitions and references.

One thought I had while looking at this ticket, though, is that the parent term, amyloid-beta clearance, seems a bit orphaned in the ontology:

GO:0008150 biological_process
-GO:0032501 multicellular organismal process
--GO:0044699 single-organism process
---GO:0044707 single-multicellular organism process
----GO:0097242 amyloid-beta clearance
-----GO:1900222 negative regulation of amyloid-beta clearance
-----GO:1900223 positive regulation of amyloid-beta clearance
-----GO:1900221 regulation of amyloid-beta clearance

I am wondering if we should consider a new grouping term in GO for 'protein homeostasis' or 'proteostasis' to house terms like 'amyloid-beta clearance'.  We could also perhaps include things like 'response to unfolded protein', etc., as additional children.  I think a term like 'protein homeostasis' would provide more biological context for the amyloid-beta clearance terms.

Looking at the existing child terms of 'chemical homeostasis' I don't see one for 'protein homeostasis', but perhaps I am missing it?

What do you think?

**Comment by vanaukenk** [comment]
On hold for just a bit - want to double-check that the distinction between parents 'cellular catabolic process' and 'catabolic process' are needed for NTRs 2 and 3.

**Comment by BarbaraCzub** [comment]
Hi @vanaukenk 
I agree that 'protein homeostasis'/'proteostasis' would be a suitable grouping term for 'amyloid-beta clearance' terms, as well as 'response to unfolded protein', etc.
Wrt 'catabolic process' it could be a parent of both NTR2 and 3. The reason why I thought about 'cellular catabolic process' for NTR2 was to attempt to specify that the catabolic process happens in a cell. But now I see that definition says 'carried our _by_ individual cells, and not _in_ individual cells, so I suppose this distinction is not really necessary, or even should be avoided.

**Comment by ValWood** [comment]
I had on my list a similar issue. I am adjusting our slim and  I have been struggling to slim annotation to

for proteins which have a role in clearing protein aggregates 
which we have currently annotated to
cellular response to misfolded protein (doesn't really capture)

AND
protein involved in protein repair.

Both of these processes could be grouped under "protein homeostasis" but maybe this is a bit to broad? Protein homeostasis  would include really  the regulation of any protein  biosynthesis in addition to protein breakdown, so it would be necessarily very high level term. Homeostasis terms are also  a bit  tricky, because you can't really be sure that the process is part of the homeostasis, or that another process is required to return the system to homeostasis.

I think you want a grouping term for the damaged protein processes?
For a while I have been wondering if we needed a term for "protein quality control" to cover these. This would allow us to begin to group the catabolic processes like ERAD which handle misfolded/damaged proteins separately from the protein catabolism that occurs to degrade proteins as they are no longer required  (for example during the cell cycle, via the APC or SCF ubiquitination pathways).

"protein quality control"
could be defined as the detection, repair or elimination of damaged proteins.

More terms belonging under such a term in this ticket.
https://github.com/geneontology/go-ontology/issues/13717

**Comment by vanaukenk** [comment]
Yes, we talked about this issue on last week's editors call, and agree that if we added a 'protein homeostasis' or 'proteostasis' term to the ontology, it would be a high level term and it might be tricky to decide exactly what child terms we would include.
Currently reading this review to try to get a better handle on this:
http://jcb.rupress.org/content/jcb/216/5/1231.full.pdf

**Comment by ValWood** [comment]
Yeah, I think we need a grouping term for the "off-pathway" OR "quality control" events in figure 1 rather then the entire protein homeostasis (which would be very difficult).

**Comment by BarbaraCzub** [comment]
Hi @vanaukenk should I reassign this ticket to myself?

**Comment by vanaukenk** [comment]
@BarbaraCzub - yes, I think that makes sense, although the utility of having a 'proteostasis' grouping term still exists, I think.  That would be a bigger project, though.
Let me know if there's anything that needs clarification here.

**Comment by BarbaraCzub** [comment]
I need to add the amyloid-beta clearance-specific terms to finish off the amyloid-beta ARUK project. I agree the 'proteostasis' grouping term would be helpful, but in terms of time I am not able to commit to  a bigger project. 

For now I'll add the new 'amyloid-beta clearance' child terms and make the changes to 'amyloid-beta clearance' definition, which I suggested in my original 19 Jul 2017 comment. 

In addition, if you agree @vanaukenk , I could make the following changes:
1. Add the terms 'proteostasis' and 'protein homeostasis' as broad (or related?) synonyms of 'GO:0019538 protein metabolic process'. 
This would be in agreement with the abstract of the paper, which you cited on 27 July 2017:
> "The proteostasis network (PN) regulates protein synthesis,
folding, transport, and degradation to maintain proteome
integrity and limit the accumulation of protein aggregates,
a hallmark of aging and degenerative diseases. In multicellular
organisms, the PN is regulated at the cellular, tissue,
and systemic level to ensure organismal health and
longevity. Here we review these three layers of PN regulation
and examine how they collectively maintain cellular
homeostasis, achieve cell type-specific proteomes, and
coordinate proteostasis across tissues. A precise understanding
of these layers of control has important implications
for organismal health and could offer new therapeutic
approaches for neurodegenerative diseases and other
chronic disorders related to PN dysfunction."
2. Make 'GO:0097242 amyloid-beta clearance' a child term of 'GO:0019538 protein metabolic process'.

Could you please let me know your thoughts @vanaukenk ? If you think this parentage change would be too drastic without updating the rest of the related ontology as a part of a bigger project, then I'll just leave the parent term, as it is for now.

cc
@RLovering 
@paolaroncaglia

**Comment by ValWood** [comment]
I changed the ticket with more detail 
https://github.com/geneontology/go-ontology/issues/13717
to mini project.
A lot of  issues in this area...

**Comment by vanaukenk** [comment]
@BarbaraCzub 
I would hold off on adding the synonyms for now, as this is a big topic and I'd want to make sure we aren't potentially directing users to a specific area of the ontology with these synonyms without first making sure we've got everything covered wrt appropriate child terms.
For your second suggestion, I just had one question about whether the transcytosis is considered a metabolic process like catabolism or rather a transport process that moves amyloid-beta from one part of the organism to another.  Looking at other protein-specific child terms of 'transcytosis', it seems they do not have 'protein metabolic process' as a parent.

**Comment by BarbaraCzub** [comment]
Hi @vanaukenk thank you for your feedback. I won't add any synonyms then. Wrt 'transcytosis', I agree it is probably more a 'transport' process. Yet, it is certainly a 'protein homeostasis' process in this case. And so the 'transport' aspect of 'transcytosis' is an additional reason not to add the 'protein homeostasis' synonyms to 'protein metabolism', as you pointed out. Thank you for your helpful comments!

So for now, I'll just do what I suggested on 19 Jul 2017, i.e.:

- [ ] add the new terms

- [ ] broaden the definition of 'GO:0097242 amyloid-beta clearance'

After I've done this, should I leave this ticket open?

**Comment by BarbaraCzub** [comment]
New terms - revisions:

I'll modify the current definition of 'amyloid-beta clearance':
> GO:0097242 'amyloid-beta clearance'
> Def.: "The process in which amyloid-beta is removed from extracellular brain regions by mechanisms involving cell surface receptors." (PMID:18289866; PMID:19098903)

I'll add _two_ new terms:
>> NTR1: 'amyloid-beta clearance by transcytosis (is_a: 'GO:0097242 amyloid-beta clearance' and is_a 'GO:0045056 transcytosis').
>> Suggested Def.: "The process in which amyloid-beta is removed from extracellular brain regions by cell surface receptor-mediated endocytosis, followed by transcytosis across the blood-brain barrier." (PMID:19098903)

>> NTR2: amyloid-beta clearance by cellular catabolic process (is_a: 'GO:0097242 amyloid-beta clearance' and is_a: 'GO:0044248 cellular catabolic process').
>> Suggested Def.: "The process in which amyloid-beta is removed from extracellular brain regions by cell surface receptor-mediated endocytosis, followed by intracellular degradation." (PMID:18289866)

**Comment by BarbaraCzub** [comment]
NTR3 is more tricky. Extracellular degradation of amyloid-beta by peptidases such as neprilysin or  insulin-degrading enzyme (IDE) (PMID-19406747, Figure 8) is not in itself a clearance process; it is only a catabolic process. In order for _clearance_ to also occur, the products of the extracellular degradation will need to get engulfed eventually.
Therefore, to truly represent what happens, instead of NTR3 'amyloid-beta clearance by extracellular catabolic process' the more accurate NTR alternatives would be:

Either:
>> NTR3a 'extracellular catabolic process involved in amyloid-beta clearance' (**part_of**: 'GO:0097242 amyloid-beta clearance' and is_a 'GO:0009056 catabolic process')
>> Suggested Def.: "The process in which amyloid-beta is degraded by peptidases present in the extracellular space as a part of its clearance." (PMID:19406747)

Or:
>> NTR3b 'amyloid-beta catabolic process' (is_a: 'GO:0050435 amyloid-beta metabolic process' and is_a 'GO:0043171 peptide catabolic process')
>> Suggested Def.: "The chemical reactions and pathways resulting in the breakdown of amyloid-beta peptides" (PMID:19406747)
(In this case an annotation with this term would require the extension: 'occurs_in: extracellular region/space', but this would be fine.)

NTR3b is a simpler term, which more accurately represents what the experiments show; however, this term would not be related to 'amyloid-beta clearance'. This would likely cause a lot of confusion to curators, therefore, I'd vote for NTR3a, even though this is a more complex term. @RLovering @paolaroncaglia @vanaukenk do you have any thoughts on this?

**Comment by paolaroncaglia** [comment]
@BarbaraCzub 

I agree with your suggested revision of the current definition of 'amyloid-beta clearance’, and with your suggested addition of NTR1 and NTR2 as above. My understanding is that those edits would not cause issues, i.e. that they’re not included in the items that the ontology group wishes to discuss more deeply.

As for NTR3, I feel that NTR3b represents the true biology better than NTR3a. If it is known that that process is *always* followed by engulfment of the catabolised parts, then you could model the new term as 'amyloid-beta catabolic process involved in amyloid-beta clearance’. But if not, my understanding is that you can’t really connect 'amyloid-beta catabolic process’ and ‘amyloid-beta clearance’ in the ontology because they don’t always occur together. If that’s the case, you may still create 'amyloid-beta catabolic process' and give it a *related* synonym of ‘amyloid-beta clearance’. 

Hope this helps, and of course happy to hear if others have different suggestions. I’ll probably have to leave this discussion here anyway, at least for the time being, as I only have a couple more days left at EBI and will then be moving/traveling/unavailable for a few weeks. 

Thanks,

Paola

**Comment by BarbaraCzub** [comment]
Should 'GO:0150094 amyloid-beta clearance by cellular catabolic process' be a descendant of 'GO:0035973 aggrephagy' and be re-named and re-defined accordingly? 

- [ ] AI: check whether this would fit with the processes described in PMID:18289866, which prompted this issue (scroll up for details).

**Comment by BarbaraCzub** [comment]
PMID:18941241

**Comment by BarbaraCzub** [comment]
Discussed the 5 Nov 2018 notes with @RLovering and decided that a new term is not required to capture the process described in these papers.
